frontends/election-manager/src/app.test.tsx

```diff
@@ -494,7 +495,7 @@ test('printing ballots and printed ballots report', async () => {
   userEvent.click(screen.getByRole('button', { name: 'Continue' }));
 
   // Printing should now continue normally
-  await waitFor(() => getByText('Printing'));
```

Comment:
In this printing modal case, we simply weren't waiting for the modal to close before moving on. Not a React Query issue.